packages/playwright-core/src/server/screenshotter.ts

```diff
@@ -138,6 +140,10 @@ export class Screenshotter {
   }
 
   async _preparePageForScreenshot(progress: Progress, disableAnimations: boolean, waitForFonts: boolean) {
+    if (disableAnimations)
+      progress.log('  disabled all CSS animations because "animations" option is set to "disabled"');
```

Comment:
`"disabled"` does not play well with languages - let's drop it? Same for `"ready"`.